### PR TITLE
Calendars+Fixed modal windows

### PR DESCRIPTION
**Pull request by Tarkeasy**
Added localization to all calendars
Made all modal windows consistent

### CODE REVIEW

File: src/app/admin-panel/admin-panel/teachers/teachers.component.scss [hunk 1]
```diff
@@ -17,13 +17,20 @@ span.ui-button-text,.ui-clickable{
 }
 .requiredWrapper{
 	position: relative;
```
Comment (dmitryy90) [inline src/app/admin-panel/admin-panel/teachers/teachers.component.scss]: indentation

File: src/app/admin-panel/admin-panel/students/students.component.ts [hunk 2]
```diff
@@ -84,9 +88,18 @@ export class StudentsComponent implements OnInit {
     this.isNew = false;
     this.showForm();
   }
-
+  formatDate(date) {
```
Comment (dmitryy90) [inline src/app/admin-panel/admin-panel/students/students.component.ts]: move to shared service

File: src/app/admin-panel/admin-panel/teachers/teachers.component.ts [hunk 3]
```diff
@@ -52,6 +54,15 @@ export class TeachersComponent implements OnInit {
     };
     reader.readAsDataURL(this.fileToUpload);
   }
+  formatDate(date) {
```
Comment (dmitryy90) [inline src/app/admin-panel/admin-panel/teachers/teachers.component.ts]: see comment above

File: src/app/admin-panel/admin-panel/teachers/teachers.component.scss (hunk 1 above)
Comment (Tarkeasy) [reply src/app/admin-panel/admin-panel/teachers/teachers.component.scss]: > indentation

fixed

File: src/app/admin-panel/admin-panel/students/students.component.ts (hunk 2 above)
Comment (Tarkeasy) [reply src/app/admin-panel/admin-panel/students/students.component.ts]: > move to shared service

done

File: src/app/admin-panel/admin-panel/teachers/teachers.component.ts (hunk 3 above)
Comment (Tarkeasy) [reply src/app/admin-panel/admin-panel/teachers/teachers.component.ts]: > see comment above

done

**Pull request merged**